src/site/xdoc/server/config-blob.xml

```diff
@@ -0,0 +1,109 @@
+
+      <subsection name="Blob Store Configuration">
+          <p>
+              <a href="https://github.com/apache/james-project/blob/master/server/blob/blob-api/src/main/java/org/apache/james/blob/api/BlobStore.java">BlobStore</a>
+              is a component used to storing blob data, it can take InputStream or byte array as the input data to store, then return the corresponding
+              <a href="https://github.com/apache/james-project/blob/master/server/blob/blob-api/src/main/java/org/apache/james/blob/api/BlobId.java">BlobId</a>.
+              Clients can use BlobStore to retrieve stored data by the BlobId.
+              <dl>
+                  <dt><strong>implementation</strong></dt>
+                  <dd>
+                      Define the BlobStore implementation will be used in the product. Available implementations: cassandra, objectstorage, union.
+                      cassandra: blobs will be stored and retrieved at cassandra database.
+                      objectstorage: blobs will be stored and retrieved at open-stack swift.
+                      union: blobs will be stored and retrieved at the current BlobStore(open-stack swift), if fails or not found, it fallback to use the legacy BlobStore (cassandra)
+                      Default is cassandra.
+                  </dd>
+              </dl>
+          </p>
+
+          <subsection name="Swift BlobStore Configuration">
+              <dl>
+                  <dt><strong>objectstorage.payload.codec</strong></dt>
+                  <dd>AES256: Aes256 encrypted blob content. DEFAULT: non encrypted</dd>
+
+                  <dt><strong>objectstorage.provider</strong></dt>
+                  <dd>Should be swift</dd>
+
+                  <dt><strong>objectstorage.namespace</strong></dt>
+                  <dd>Namespace to store blobs in</dd>
+
+                  <dt><strong>objectstorage.swift.authapi</strong></dt>
+                  <dd>Should be tmpauth</dd>
+
+                  <dt><strong>objectstorage.swift.endpoint</strong></dt>
+                  <dd>Swift authentication endpoint. Should be http://swift-host-address:8080/auth/v1.0 </dd>
+
+                  <dt><strong>objectstorage.swift.credentials</strong></dt>
+                  <dd>Should be testing</dd>
+
+                  <dt><strong>objectstorage.swift.tempauth.username</strong></dt>
+                  <dd>Should be tester</dd>
+
+                  <dt><strong>objectstorage.swift.tempauth.tenantname</strong></dt>
+                  <dd>Should be test</dd>
+
+                  <dt><strong>objectstorage.swift.tempauth.passheadername</strong></dt>
+                  <dd>Should be X-Storage-Pass</dd>
+
+                  <dt><strong>objectstorage.swift.tempauth.passheadername</strong></dt>
+                  <dd>Should be X-Storage-User</dd>
+              </dl>
+          </subsection>
+      </subsection>
+
+      <subsection name="Blob Export Configuration">
```

Comment:
I don't agree that Blob Export should be linked in any way to Blob Store. It shouldn't be linked in the code nor in the documentation.

Reply:
No, it's not in the same section. I placed blob store and blob export in two distinct section. There is no link between them.

Reply:
still it's in the Blob Store configuration documentation. It's not ok.

Reply:
Okay, I separated blob export configuration to a distinct file